orchestrator/nodes/notification/notification.html

```diff
@@ -0,0 +1,44 @@
+<script type="text/x-red" data-template-name="notification">
+    <div class="form-row">
+        <label for="node-input-name"><i class="fa fa-tag"></i> <span data-i18n="label.name"></label>
+        <input type="text" id="node-input-name">
+    </div>
+    <div class="form-row">
+        <label for="node-input-name"><i class="fa fa-edit"></i> <span data-i18n="label.message"></label>
+        <input type="text" id="node-input-message_notification">
+    </div>
+        <div class="form-row">
+        <label for="node-input-name"><i class="fa fa-cube"></i> <span data-i18n="label.source"></label>
```

Comment:
`source` refers to `metaAttrsFilter` seems a little confusing. I don't know if it's clear to the user, maybe `metadata` sounds better